internal/docker/config.go

```diff
@@ -20,12 +25,12 @@ type Config struct {
 	ExcludedImages []string `mapstructure:"excluded_images"`
 
 	// Docker client API version.
-	DockerAPIVersion float64 `mapstructure:"api_version"`
+	DockerAPIVersion APIVersion `mapstructure:"api_version"`
```

Comment:
just so I understand - we expose this with `APIVersion`, is there a reason not to use string instead?

Reply:
Ok, updated to use strings throughout, which simplifies the internal helpers a bit since we aren't relying on an unmarshalling implementation in config loading.